<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PMH::Internal::PmhPreferencesWidget</class>
 <widget class="QWidget" name="PMH::Internal::PmhPreferencesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Data saving</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="deletionBox">
        <property name="text">
         <string>Always ask for confirmation before deletion</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Category background color</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Utils::ColorButtonChooser" name="categoryBackgroundButton">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Category background color</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Category foreground color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Utils::ColorButtonChooser" name="categoryForegroundButton">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Category foreground color</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="toolTip">
         <string>PMHx background color</string>
        </property>
        <property name="text">
         <string>PMHx background color</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Utils::ColorButtonChooser" name="pmhBackgroundButton">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>PMHx foreground color</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="Utils::ColorButtonChooser" name="pmhForegroundButton">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="toolTip">
         <string>PMHx foreground color</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Fonts</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Category font</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="changeCatFont">
        <property name="text">
         <string>Change...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>PMHx font</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="changePmhFont">
        <property name="text">
         <string>Change...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>162</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::ColorButtonChooser</class>
   <extends>QPushButton</extends>
   <header location="global">utils/widgets/colorbuttonchooser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
